database/seedGenerator.test.js

```diff
@@ -0,0 +1,55 @@
+const generateAllSeedData = require('./seedingUtils.js');
+
+describe('Seed Generation', () => {
+  describe('seed generation function', () => {
+    test('Creates one set of User and Project data', () => {
+      const single = generateAllSeedData(1);
+      expect(single.users.length).toBe(1);
+      expect(single.projects.length).toBe(1);
+    });
+
+    test('Creates multiple updates', () => {
+      const single = generateAllSeedData(1);
+      expect(single.updates.length).toBeGreaterThan(0);
+    });
+
+    test('Can create several sets of data', () => {
+      const SEED_SIZE = 10;
+      const several = generateAllSeedData(SEED_SIZE);
+      expect(several.users.length).toBe(SEED_SIZE);
+      expect(several.projects.length).toBe(SEED_SIZE);
+      expect(several.updates.length).toBeGreaterThan(SEED_SIZE);
```

Comment:
Again, this will pass with only 1 update per set.

Reply:
I don't believe this one will pass with only 1 update per set because the last line there is checking to ensure that the number of updates is greater than the number sets made.